### DISCUSSION TITLE
[質問]:  TableViewのBarButtonItemがずれてしまう＋上に謎の空白ができてしまう

### DISCUSSION THREAD

**Discussion by sheep1857** [lit-iphone-course/lit-iphone-course#151] [Question-original] [answered]
### エラーの概要

TableViewのBarButtonItemが文字数が多いからなのか、実機で実行するとずれてしまいます。
また、Barの上に空白ができてしまい、直そうとしましたがSafeAreaがそれ以上上に行かないため解決できません。

![IMG_6011](https://user-images.githubusercontent.com/105029351/226102992-7c74df26-826a-402d-8982-906db2bced25.PNG)

### GitHubのURL

https://github.com/sheep1857/GoshuinApp.git

### TODOリスト

- [X] GitHubに最新のコードをアップロードしました。
- [X] できるだけ具体的なタイトルを設定しました。

**Comment by txxxxc** [comment, marked as answer]
原因はNavigationBarがViewの下に配置されている部分です！
![image](https://user-images.githubusercontent.com/22112440/226104308-0319f7b6-9748-4e83-9d31-c190b01f85a6.png)

解決策としては上記で指し示しているNavigationBarを削除して、リストから探すViewControllerからSegueを設定しなおすことで直るかと思います！

試してみてくださいー！

**Reply by sheep1857** [reply]
今、リストから探すVCにあるプラスボタンからSegueを繋いでいるのですが、設定し直すというのはそこを繋ぎ直せばいいんですか？

**Reply by txxxxc** [reply]
>設定し直すというのはそこを繋ぎ直せばいいんですか

そうです！今設定してるSegueを削除して、もう一度設定し直してみてくださいー！

**Reply by sheep1857** [reply]
できました！ありがとうございます！
